src/reynir_correct/errtokenizer.py

```diff
@@ -2721,11 +2775,13 @@ def final_correct(self, stream: TokenIterator) -> TokenIterator:
         # as numbers ('24 Milljónir') and amounts ('3 Þúsund Dollarar')
         ct_stream = cast(Iterator[CorrectToken], stream)
         token_ctor = cast(TokenCtor, self._token_ctor)
-        return cast(
-            TokenIterator,
-            late_fix_capitalization(ct_stream, self._db, token_ctor, self._only_ci),
+        ct_stream = late_fix_capitalization(
+            ct_stream, self._db, token_ctor, self._only_ci
         )
 
+        ct_stream = late_fix_merges(ct_stream, self._db, token_ctor, self._only_ci)
```

Comment:
Delete the db, token_ctor and only_ci parameters here

Reply:
Done